docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,225 @@
+
+```sh
+testcafe --help
+```
+
+### <a class="anchor" name="b"></a>-b, --list-browsers 
+Lists [aliases] of the browsers available on this computer.
+
+```sh
+testcafe --list-browsers
+```
+
+### -r \<name\>, --reporter \<name\>  
+Specifies a built-in or custom reporter that will be used to generate test reports. 
+The available built-in reporters are [json](https://github.com/DevExpress/testcafe-reporter-json), 
+[list](https://github.com/DevExpress/testcafe-reporter-list), 
+[minimal](https://github.com/DevExpress/testcafe-reporter-minimal), 
+[spec](https://github.com/DevExpress/testcafe-reporter-spec), 
+[xunit](https://github.com/DevExpress/testcafe-reporter-xunit). 
+By default, the `spec` reporter is used.
+
+The following command runs tests in all available browsers and generates a report in the xunit format.
+
+```sh
+testcafe all /tests/sample-fixture.js -r xunit
+```
+
+The following command runs tests and generates a test report by using the custom plugin reporter.
+
+```sh
+testcafe all /tests/sample-fixture.js -r my-reporter
+```
+
+A generated test report will be displayed in the command prompt window. 
+
+If you need to save a report to an external file, you can redirect the command output stream to a file  
+by using the > redirection operator and specifying a report file path.
+
+```sh
+testcafe all /tests/sample-fixture.js > /tests/test-results.txt
+```
+
+### -s \<path\>, --screenshots \<path\> 
+Enables screenshot capturing and specifies the directory path
+to save the screenshots to.
+
+```sh
+testcafe all /tests/sample-fixture.js -s /screenshots
+```
+
+### -S, --screenshots-on-fails
+Takes a screenshot whenever a test fails. Screenshots are saved to the directory
+specified by using the **-screenshots \<path\>** option.
+
+For example, the following command runs tests from the
+  *sample-fixture.js* file in all browsers, takes screenshots if tests fail
+  and saves the screenshots to the */screenshots* directory.
+
+```sh
+testcafe all /tests/sample-fixture.js -S -s /screenshots
+```
+
+### -q, --quarantine-mode 
+Enables the quarantine mode for tests that fail. 
+In this mode, a failed test is executed several more times. 
+The test result depends on the outcome (*passed* or *failed*) that will be at most. 
+That is, if the test fails on most attempts, the result is *failed*. 
+And vice versa, if the test is executed successfully in most of attempts, the result is *passed*.
+
+If the test result differs between runs, the test is marked as unstable.
+
+```sh
+testcafe all /tests/sample-fixture.js -q
+```
+
+### -e, --skip-js-errors
+Makes tests not fail when a JavaScript error occurs on a tested web page.
+By default, in case of such errors, TestCafe stops test execution and posts
+an error message to a report. If you want TestCafe to ignore JavaScript errors,
+specify this argument.
+
+For example, the following command runs test from the specified file and forces TestCafe to ignore JavaScript errors.
```

Comment:
I'll change to "runs tests". We don't know which tests are there, we only know the name of file.